pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge PDF:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
+
+- Separate PDF into individual pages:
+
+`pdftk {{input.pdf}} burst`
+
+- Rotate third page by 90 degrees clockwise:
```

Comment:
You should also mention that this is picking a set of pages. Or even better: make this an example of copying all pages from a source pdf, but rotating the third one.